.travis.yml

```diff
@@ -71,14 +71,17 @@ jobs:
       name: "Node 2"
       env: KNAPSACK_PRO_CI_NODE_INDEX=2
     - stage: Test
-      name: "Front-end tests, Storybook, bundle audit, console check, e2e"
+      name: "Front-end tests, Storybook, bundle audit, console check"
       script:
         - yarn lint:frontend
         - yarn test --colors
         - bundle exec rails db:create db:schema:load assets:precompile
         - bundle exec bundle-audit check --update
         - bin/test-console-check
         - yarn build-storybook
+    - stage: test
+      name: "E2E"
+      script:
```

Comment:
fairly certain you'll want/need the rails db:schema:load and assets:precompile before running e2e here

Reply:
Ah right. I'm facing another issue where Travis refuse to start builds